tests/integration_tests/test_game.py

```diff
@@ -532,3 +537,112 @@ async def test_gamestate_ended_clears_references(
 
     assert test.lobby_connection.game_connection is None
     assert rhiza.lobby_connection.game_connection is None
+
+
+@pytest.mark.rabbitmq
+@fast_forward(30)
+async def test_game_ended_broadcasts_army_results(lobby_server, channel):
```

Comment:
We should definitely have a test for a game with uneven teams, like 2v1 for example. Dunno if you wanna do 2 separate tests or just make this one use uneven teams.